### PR TITLE
Bug/upload match

### PR DESCRIPTION
**Pull request by ab7289**
This fixes a bug when uploading apartments.

If there is an existing location with city of say, "New York" uploading an apartment with city "new York" would create a new location instead of adding the apartment to the existing location. This fixes that issue so capitalization differences on the city do not create new locations.

### CODE REVIEW

File: location/views.py [hunk 1]
```diff
@@ -397,11 +397,12 @@ def apartment_upload(request):
             # parse out the latitude and longitude from the response
             lat_lng = g_utils.parse_lat_lng(g_data[0])
 
-            g_locality = g_utils.get_city(g_data[0])[1]
+            g_city, g_locality = g_utils.get_city(g_data[0])
+            street_num, street = g_utils.get_address(g_data[0])
```
Comment (KShih) [inline location/views.py]: image
Not sure if it is within this PR's scope.
But if you add the `state` and `city` in the address filed, it created a location with duplicate state and city.

File: location/views.py [hunk 2]
```diff
@@ -397,11 +397,12 @@ def apartment_upload(request):
...
```
Comment (KShih) [inline location/views.py]: image
Expected: 219 east 32nd street == 219 East 32nd Street

File: location/views.py (hunk 1 above)
Comment (ab7289) [reply location/views.py]: fixed

File: location/views.py (hunk 2 above)
Comment (ab7289) [reply location/views.py]: fixed

Comment (KShih) [reply location/views.py]: Nice!

File: location/views.py (hunk 1 above)
Comment (KShih) [reply location/views.py]: Nice

**Pull request merged**